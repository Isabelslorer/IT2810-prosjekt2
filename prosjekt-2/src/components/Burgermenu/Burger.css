.burgerBar {
    transition: 0.3s; 
    margin-top: 2.5px;
    margin-bottom: 2.5px;
    background-color: beige;
    margin: 5px;
    width: 40px;
    height: 5px;
  }

  .burgermenuContainer:hover {
    cursor: pointer;
}

  @media only screen and (max-width: 600px) {
    .burgerBar{
        width: 25px;
        height: 3px;
        margin: 3px;
    }
  }