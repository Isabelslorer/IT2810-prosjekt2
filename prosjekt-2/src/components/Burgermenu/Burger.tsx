import React from 'react';
import './Burger.css';

interface Props{
    setOpen: (val: boolean) => void;
    isOpen: boolean;
}

export default function Burger(props: Props) {
  return (
    <div className="burgermenuContainer" onClick={() => props.setOpen(!props.isOpen)}>
        <div className="burgerBar"></div>
        <div className="burgerBar"></div>
        <div className="burgerBar"></div>
    </div> 
  );
}



