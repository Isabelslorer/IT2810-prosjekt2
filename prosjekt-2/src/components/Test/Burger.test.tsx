import React from 'react';
import renderer from 'react-test-renderer';

import Burger from '../../components/Burgermenu/Burger';

it("renders correctly when there are no items", () => {
    const tree = renderer.create(<Burger isOpen={false} setOpen={() => {}}/>).toJSON();
    expect(tree).toMatchSnapshot();
});