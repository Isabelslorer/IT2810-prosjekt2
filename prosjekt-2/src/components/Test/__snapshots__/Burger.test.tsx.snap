// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="burgermenuContainer"
  onClick={[Function]}
>
  <div
    className="burgerBar"
  />
  <div
    className="burgerBar"
  />
  <div
    className="burgerBar"
  />
</div>
`;
